src/pages/cart/index.tsx

```diff
@@ -1,5 +1,42 @@
-import { ReactElement } from 'react';
+import { ReactElement, useContext } from 'react';
+import { CartItem } from '../../components/general/cart-item';
+import { CartContext } from '../../contexts/Cart';
 
 export const CartPage = (): ReactElement => {
-  return <div>Cart Page</div>;
+  const { cart, setCart } = useContext(CartContext);
+  type Amount = 'increment' | 'decrement';
+  const changeQuantity = (id: string, amount: Amount) => {
```

Comment:
Nice you solved the double add! :D 

Not sure if this is relevant for your ticket so maybe ignore it, but could you extract the handler functions out to a separate file? I think my ticket will need access to them for the cart list in the header and I think we’ll need changeQuantity for the Add To Cart button too?

Reply:
yeah i just put temporally added to the cart page till you was done!

Reply:
fab, you're one step ahead of me 😅😅

Reply:
you could create an utils folder and add helpers there